SimTracker/TrackerHitAssociation/src/TrackerHitAssociator.cc

```diff
@@ -114,7 +114,7 @@ void TrackerHitAssociator::makeMaps(const edm::Event& theEvent, const TrackerHit
           theEvent.labelsForToken(cfToken, labels);
           std::string trackerContainer(labels.productInstance);
```

Comment:
@makortel But is it not so that even if I move the comparison out of the loop over *thisContainerHits, I still neet to look at every element of a cfToken or rather thisContainerHits.

So I am not sure about the advantage or maybe I have not quite understand your suggestion.